binding.gyp

```diff
@@ -14,6 +14,7 @@
           'libraries': [
             '<(PRODUCT_DIR)/../../windows/lib/libzmq',
             'ws2_32.lib',
+            'iphlpapi'
```

Comment:
lol. IP Helper API. https://msdn.microsoft.com/en-us/library/windows/desktop/aa366073(v=vs.85).aspx

A good example of how adding an 'e' to make a human readable name would have been a solid idea. `iphelpapi` would have been better than `iphlpapi`.

Overall, I think the addition here is fine.